src/epics/orgUnits.js

```diff
@@ -37,7 +37,7 @@ export const loadOrgUnitTree = action$ =>
         getD2()
             .then(d2 =>
                 d2.models.organisationUnits.list({
-                    level: 1,
+                    userDataViewFallback: true,
```

Comment:
What does this change do?

Reply:
Fixes a bug reported by @janhenrikoverland. There might be users that don't have access to level 1 in the org unit hierarchy, and this query should only return the topmost the user has access to. Added a comment in the code explaining it. This fix was also backported to 2.30 and 2.29.